family2family/src/main/resources/static/js/admin.jsx

```diff
@@ -0,0 +1,218 @@
+            <User key={user.id} user={user}/>
+        );
+        return (
+            <div>
+                <div className="row">
+                    <div className="col-xs-3">
+                        User Id
+                    </div>
+                    <div className="col-xs-3">
+                        User Name
+                    </div>
+                    <div className="col-xs-3">
+                        Administrator
+                    </div>
+                    <div className="col-xs-3">
+                        Active
+                    </div>
+                </div>
+                {users}
+            </div>
+        )
+    }
+}
+
+class NavBarComponent extends React.Component {
+    render() {
+        return(
+            <nav className="navbar navbar-default">
+                <div className="container-fluid">
+                    <ul className="nav navbar-nav">
+                        <li className="active"><a href="#">Home</a></li>
+                        <li><a href="#Page1">Page 1</a></li>
+                        <li><a href="#Page2">Page 2</a></li>
+                        <li><a href="#Page3">Page 3</a></li>
+                    </ul>
+                </div>
+            </nav>
+        )
+    }
+}
+
+class TabContentFrames extends React.Component {
+    render() {
+        return (
+            <div className="container">
+                <div className="usersFrame">
+                    <UserTable />
+                </div>
+                <div className="refTablesFrame">
+                    <h1>This is the ref table tab</h1>
+                </div>
+            </div>
+        )
+    }
+}
+
+class TabBarComponent extends React.Component {
+
+    displayUsers(e){
+        e.preventDefault();
+        console.log("Users Frame");
+        document.getElementsByClassName("refTablesFrame")[0].style.display = 'none';
+        document.getElementsByClassName("usersFrame")[0].style.display = 'block';
+    }
+
+    displayRefTable(e) {
+        e.preventDefault();
+        console.log("RefTable");
+        document.getElementsByClassName("usersFrame")[0].style.display = 'none';
+        document.getElementsByClassName("refTablesFrame")[0].style.display = 'block';
+    }
+
+    render() {
+        return(
+            <div className="container">
+                <ul className="nav nav-tabs">
+                    <li><a onClick={this.displayUsers.bind(this)}>Users</a></li>
+                    <li><a onClick={this.displayRefTable.bind(this)}>Reference Tables</a></li>
+                </ul>
+                <TabContentFrames />
+            </div>
+        )
+    }
+}
+
+class EditUserRow extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {id:props.id, firstName:props.firstName, lastName:props.lastName, admin:props.admin, active:props.active};
+    }
+
+    handleChange(event) {
+        this.setState({value: event.target.value});
```

Comment:
I created https://github.com/coop-code-challenge/cbtf/issues/105 to fix this one later.